### PR TITLE
merge 80 fixes for Case 21388, Case 21387, Case 21375

### PR DESCRIPTION
**Pull request by roxanneskelly**
Case 21388, Case 21387, Case 21375

Case 21388- Clicking on item picture from Inventor Items list does not
always display the Marketplace item page.
Case 21387- See All Markets page is missing its footer and the two
Explore buttons no longer work
Case 21375- QML Marketplace - For unavailable items (no more editions
for sale), marketplace item list 'buy' button is not disabled

Also, a few minor font sizing and other graphics tweaks

https://highfidelity.manuscript.com/f/cases/21388/Clicking-an-item-picture-from-the-Inventory-Items-list-does-not-always-display-the-Marketplace-item-page

https://highfidelity.manuscript.com/f/cases/21387/See-All-Markets-page-is-missing-its-footer-and-the-two-Explore-buttons-no-longer-work

https://highfidelity.manuscript.com/f/cases/21375/v0-81-0-QML-Marketplace-For-unavailable-items-no-more-editions-for-sale-marketplace-item-list-buy-button-is-not-disabled

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17536/HighFidelity-Beta-PR14997-6a63f85.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17536/HighFidelity-Beta-PR14997-6a63f85.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17536/HighFidelity-Beta-Interface-PR14997-6a63f85.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17536/HighFidelity-Beta-PR14997-6a63f85.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17536/HighFidelity-Beta-Interface-PR14997-6a63f85.exe

File: interface/resources/qml/hifi/commerce/marketplace/MarketplaceListItem.qml [hunk 1]
```diff
@@ -298,19 +299,21 @@ Rectangle {
...
                     bottomMargin: 10
                 }
+                width: 180
 
                 property bool isNFS: availability === "not for sale" // Note: server will say "sold out" or "invalidated" before it says NFS
                 property bool isMine: creator === Account.username
                 property bool isUpgrade: root.edition >= 0
                 property int costToMe: ((isMine && isNFS) || isUpgrade) ? 0 : price
-                property bool isAvailable: costToMe >= 0
+                property bool isAvailable: availability === "available"
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/commerce/marketplace/MarketplaceListItem.qml]: why this change? doesn't that break the ability of the creator to stock stuff

Comment (roxanneskelly) [reply interface/resources/qml/hifi/commerce/marketplace/MarketplaceListItem.qml]: Maybe I'm not understanding the creator of stock stuff correctly.  The way it was broke the 'sold out' and 'invalidated' cases, leaving the button active instead of greyed out, and leaving the text containing the price.

Comment (roxanneskelly) [reply interface/resources/qml/hifi/commerce/marketplace/MarketplaceListItem.qml]: ah, I think I understand now.  Will fix.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17588/HighFidelity-Beta-PR14997-9aed9b5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17588/HighFidelity-Quest-Beta-PR14997-9aed9b5.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17588/HighFidelity-Beta-PR14997-9aed9b5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17588/HighFidelity-Beta-Interface-PR14997-9aed9b5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17588/HighFidelity-Beta-PR14997-9aed9b5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17588/HighFidelity-Beta-Interface-PR14997-9aed9b5.exe

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by howard-stearns** [comment]
Referring to the comment from "gustavo", this probably won't happen for at least a week. Critical path is 80, which is has it's own issues. And master has deviated from 80 on the things here. Once we get 80 straightened out, we will untangle this as well.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17953/HighFidelity-Beta-PR14997-d611bfa.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17953/HighFidelity-Beta-Interface-PR14997-d611bfa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17953/HighFidelity-Beta-PR14997-d611bfa.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17953/HighFidelity-Beta-Interface-PR14997-d611bfa.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17953/HighFidelity-Beta-PR14997-d611bfa.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17953/HighFidelity-Quest-Beta-PR14997-d611bfa.apk)
